network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -498,39 +632,16 @@ def categories_page(self, request, slug):
         except BuyersGuideProductCategory.DoesNotExist:
             category = get_object_or_404(BuyersGuideProductCategory, name__iexact=slug)
 
-        # TODO
-        # def sort_on_creepiness(product_set):
-        #     return sorted(product_set, key=get_average_creepiness)
-        #
-        # Must sort pages by their creepiness levels
-        # key = f'products_category__{slug.replace(" ", "_")}'
-        # products = cache.get_or_set(
-        #     key,
-        #     lambda: sort_on_creepiness(
-        #         [p.to_dict() for p in Product.objects.filter(product_category__in=[category]).distinct()]
-        #     ),
-        #     86400
-        # )
-        products = []
-
-        # def filter_draft_products(request, products):
-        #     if request.user.is_authenticated:
-        #         return products
-
-        #     return filter(lambda p: p['draft'] is False, products)
-
-        # products = filter_draft_products(request, products)
-
-        return render(request, 'category_page.html', {
-            'pagetype': 'category',
-            'categories': BuyersGuideProductCategory.objects.all(),
-            'category': category,
-            'products': products,
-            'mediaUrl': MEDIA_URL,
-            'pageTitle': pgettext(
+        products = ProductPage.objects.filter(product_categories__category__in=[category]).live()
+        products = sorted(products, key=lambda p: p.average_vote_count, reverse=True)
```

Comment:
We may want to make this its own function that can be imported as `from ProductPage import sort_average` or the like, rather than hardcoding it in a route. Also, why are we reversing? The sorting order on the homepage and category page is the same.

Reply:
Oh I was playing with reverse to make sure lambda was sorting the way I expected. It shouldnt be reversed. Glad you caught this. 

Right, Ill move this to its own function. 👍